domain/domain.go

```diff
@@ -255,19 +256,26 @@ func (do *Domain) tryLoadSchemaDiffs(m *meta.Meta, usedVersion, newVersion int64
 		diffs = append(diffs, diff)
 	}
 	builder := infoschema.NewBuilder(do.infoHandle).InitWithOldInfoSchema()
-	tblIDs := make([]int64, 0, len(diffs))
+	phyTblIDs := make([]int64, 0, len(diffs))
+	actions := make([]uint64, 0, len(diffs))
 	for _, diff := range diffs {
-		ids, err := builder.ApplyDiff(m, diff)
+		IDs, err := builder.ApplyDiff(m, diff)
 		if err != nil {
 			return false, nil, err
 		}
 		if canSkipSchemaCheckerDDL(diff.Type) {
 			continue
 		}
-		tblIDs = append(tblIDs, ids...)
+		phyTblIDs = append(phyTblIDs, IDs...)
+		for i := 0; i < len(phyTblIDs); i++ {
```

Comment:
maybe this should be  `i := lastPhyTblIDsLen; i < ....`? :confused:

Reply:
This should be `i < len(IDs)`,  it's changed incorrectly by refactor